### ISSUE TITLE
Fake "Homepage Focus Areas and Spotlights" data generation raises errors

### ISSUE THREAD

**Issue by mmmavis** [mozilla/foundation.mozilla.org#5139]
### Describe the bug

Errors raised when generating fake data for `Homepage Focus Areas and Spotlights`

```bash
Generating Homepage Focus Areas and Spotlights
Traceback (most recent call last):
  File "network-api/manage.py", line 34, in <module>
    execute_from_command_line(sys.argv)
  File "/app/dockerpythonvenv/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/app/dockerpythonvenv/lib/python3.7/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/app/dockerpythonvenv/lib/python3.7/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/app/dockerpythonvenv/lib/python3.7/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/app/network-api/networkapi/management/commands/load_fake_data.py", line 77, in handle
    mozfest_factory
  File "/app/network-api/networkapi/management/commands/load_fake_data.py", line 70, in <listcomp>
    [app_factory.generate(seed) for app_factory in [
  File "/app/network-api/networkapi/wagtailpages/factory/__init__.py", line 26, in generate
    homepage_features.generate(seed)
  File "/app/network-api/networkapi/wagtailpages/factory/homepage_features.py", line 25, in generate
    area=FocusArea.objects.get(name='Empower Action')
  File "/app/dockerpythonvenv/lib/python3.7/site-packages/django/db/models/manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/app/dockerpythonvenv/lib/python3.7/site-packages/django/db/models/query.py", line 408, in get
    self.model._meta.object_name
networkapi.wagtailpages.pagemodels.base.DoesNotExist: FocusArea matching query does not exist.
```

**Comment by cadecairos** [comment]
in Django shell, the name attibute of FocusArea models is coming back blank:

```
>>> FocusArea.objects.values()
<MultilingualQuerySet [{'id': 1, 'page_id': None, 'interest_icon_id': None, 'description': '', 'name': ''}, {'id': 2, 'page_id': None, 'interest_icon_id': None, 'description': '', 'name': ''}, {'id': 3, 'page_id': None, 'interest_icon_id': None, 'description': '', 'name': ''}]>
```

**Comment by cadecairos** [comment]
there's a specific django-modeltranslation step that must be run (inv l10n-update), because of the order of operations we're doing.

1. we run a data migration to add default focus areas
2. we immediately migrate them to use django-modeltranslation

Without that update step, modeltranslation doesn't go in and grab "name" and "description" to populate "name_en" and "description_en" which the "MultilingualQuerySet" draws from

PR incoming

**Comment by cadecairos** [comment]
fixed in #5173
